enderio-conduits/src/main/java/crazypants/enderio/conduits/conduit/liquid/EnderLiquidConduitNetwork.java

```diff
@@ -44,6 +47,11 @@ public void connectionChanged(@Nonnull EnderLiquidConduit con, @Nonnull EnumFaci
     tankMap.remove(key);
     tankMap.put(key, tank);
 
+    hasMultipleTanksSet.remove(key);
+    if (tank.isValid() && !tank.externalTank.getTankInfoWrappers().isEmpty()) {
```

Comment:
Would `.size() > 1` instead of `!.isEmpty()` make sense here? Do you think assuming that a tank that has a tank list of 1 will stay at 1 and return that one tank's content when asked to "anything"?

Reply:
It may be a bit paranoid, but the idea here was that `drain` might access a different tank that isn't exposed by `getTankInfoWrappers`.

That is quite unlikely though, so I've changed it to `.size() > 1` in 4f71eaa

Reply:
yes, I was thinking about that, too. But the chance that a single-tank block correctly implements the properties is much, much higher.